app/Services/Manager.php

```diff
@@ -37,9 +47,9 @@ public function exportCSV($model, $reportbacks = false)
         $users = $model->users;
 
         $headers = ['northstar_id', 'first_name', 'last_name', 'email', 'cell'];
+
         if ($reportbacks) {
```

Comment:
its just an indicator that I created in a previous PR so we know whether the CSV should add reportback data or not. (so just a boolean, no reportback data in it)